source/ns_cmdline.c

```diff
@@ -1892,3 +2101,47 @@ char *cmd_parameter_last(int argc, char *argv[])
     }
     return NULL;
 }
+static const char* strrevchr(const char* from, const char* to, const char c)
```

Comment:
What exactly is this function meant to do? Find the last occurrence of a character in a string?

Reply:
exactly - between two pointers